### DISCUSSION TITLE
Airbyte S3 Source Setup: Need to set datetime data type for columns in JSON SCHEMA

### DISCUSSION THREAD

**Discussion by sunila96** [json-schema-org/community#569] [Q&A]
Hi everyone, I am working on airbyte where I created an S3 source and I have to pull data into redshift, my requirements are to pull data into redshift following the same datatypes. In my S3 files, I have date values which need to be pulled in the same format but JSON Schema does not have a date type. Is there an alternate datatype that can be used here?

![image](https://github.com/json-schema-org/community/assets/141052884/9590ea87-02b3-4b41-ad50-747e786eb215)

**Comment by jdesrosiers** [comment]
JSON Schema can only describe JSON type, which doesn't include dates. Types that aren't supported by JSON are typically represented as strings and further classified using the [`format`](https://json-schema.org/understanding-json-schema/reference/string#format) keyword.

```json
{
  "type": "string",
  "format": "date"
}
```

In the category of dates, JSON Schema defines "date", "time", "date-time", and "duration".
